admin_manual/configuration_server/occ_command.rst

```diff
@@ -1146,6 +1146,10 @@ This example restores the deleted files of all users::
 This example restores the deleted files of users molly and freda::
 
  sudo -u www-data php occ trashbin:restore molly freda
+ 
+.. note::
+  This command only restores Users deleted files.
+  If you're using Group Folders app, such files are not restored.
```

Comment:
Is this intentional BTW?
Is there another command to restore groupfolder deleted files?

Reply:
I don't think so, but yeah, this is a problem beacause ATM only WebUI can restore files (and browser == timeouts for big Trashbin) 😢